examples/typescript/backends/cloud/cdktf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "language": "typescript",
+  "app": "npx ts-node main.ts",
+  "terraformProviders": [
+    "aws@~> 3.0"
```

Comment:
let's use a smaller one (as it is not used anyway).
wait.. I think we can also remove it completely as `cdktf get` does not fail anymore if there is nothing to do, right?